packages/components/src/FilterSelect/types.ts

```diff
@@ -1,34 +1 @@
-import { Node } from "@react-types/shared"
-
-export type SelectOption = {
-  label: string
-  value: React.Key
-  disabled?: boolean
-  options?: never
-}
-
-export type SelectOptionGroup<Option extends SelectOption> = {
-  label: string
-  options: Iterable<Option>
-}
-
-export type SelectItem<Option extends SelectOption = SelectOption> =
-  | Option
-  | SelectOptionGroup<Option>
-
-export type SelectOptionNode<Option extends SelectOption = SelectOption> =
-  Node<Option> & {
-    type: "item"
-    hasChildNodes: false
-  }
-
-export type SelectOptionGroupNode<Option extends SelectOption = SelectOption> =
-  Omit<Node<SelectOptionGroup<Option>>, "childNodes"> & {
-    type: "section"
-    hasChildNodes: true
-    childNodes: Array<SelectOptionNode<Option>>
-  }
-
-export type SelectItemNode<Option extends SelectOption = SelectOption> =
-  | SelectOptionNode<Option>
-  | SelectOptionGroupNode<Option>
+export * from "~components/__future__/Select/types"
```

Comment:
For consumers. Otherwise if they want to use the types they'll have to dig into `future` and that would be pretty weird given `FilterSelect` isn't from there 😅 

I can leave a comment though. This is only temporary until they aren't split.

Reply:
Oh, I didn't see that these types are being made available to consumers, cool cool